container.go

```diff
@@ -36,6 +36,7 @@ var containerCommand = &cli.Command{
 		startCommand,
 		killCommand,
 		pauseCommand,
+		containerWaitCommand,
```

Comment:
waitCommand can be used directly

Reply:
I know but we have too docker container wait. Should i ignore it ?

Reply:
Please see other commands such as pauseCommand, and follow the coding style

Reply:
Yep I see